### ISSUE TITLE
MissingPluginException (No implementation found for method getAll)

### ISSUE THREAD

**Issue by Shankar-Joshi** [parse-community/Parse-SDK-Flutter#712]
### New Issue Checklist

- [ !] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [! ] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [ !] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).
- [! ] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Flutter SDK](https://github.com/parse-community/Parse-SDK-Flutter/releases). 

### Issue Description

I am using parse_server_sdk_flutter: ^3.1.0 in flutter but I am getting error as below: 

[ERROR:flutter/lib/ui/ui_dart_state.cc(198)] Unhandled Exception: MissingPluginException(No implementation found for method getAll on channel dev.fluttercommunity.plus/package_info)
#0      MethodChannel._invokeMethod
<asynchronous suspension>
#1      MethodChannel.invokeMapMethod
<asynchronous suspension>
#2      MethodChannelPackageInfo.getAll
<asynchronous suspension>
#3      PackageInfo.fromPlatform
<asynchronous suspension>
#4      main
<asynchronous suspension>
Restarted application in 441ms. 

It is throwing error at this function:

void main() async {
  WidgetsFlutterBinding.ensureInitialized();

  PackageInfo packageInfo = await PackageInfo.fromPlatform();

  late final keyApplicationId = 'myappID';
  late final keyClientKey = 'clientkey';
  late final keyParseServerUrl = 'http://localhost:1337/parse';

  await Parse().initialize(keyApplicationId, keyParseServerUrl,
      clientKey: keyClientKey, debug: true);

  runApp(MaterialApp(
    home: Home(),
  ));
}

### Steps to reproduce

Follow the current parse_server_sdk_flutter: ^3.1.0 docs and run app in windows. 

### Actual Outcome

I have tried everything that I found in google trying to troubleshoot but nothing working. 

### Environment

Flutter for windows app

Parse Flutter SDK
- SDK version: `arse_server_sdk_flutter: ^3.1.0`
- Operating system version: `windows 10`

Server
- Parse Server version: `[5.2.0]`

### Logs

 if (result == null) {
      if (missingOk) {
        return null;
      }
      throw MissingPluginException('No implementation found for method $method on channel $name');

Exception has occurred.
MissingPluginException (MissingPluginException(No implementation found for method getAll on channel dev.fluttercommunity.plus/package_info))

**Comment by RodrigoSMarques** [comment]
The package is in the process of being updated and will be released in the next few days.

Please test and get back if it's ok.

In `pubspec.yaml`, make the changes below.

```dart
  #parse_server_sdk_flutter: ^3.1.0
  parse_server_sdk_flutter:
    git:
      url: https://github.com/RodrigoSMarques/Parse-SDK-Flutter.git
      ref: development
      path: packages/flutter

**Comment by mbfakourii** [comment]
Solved via [#762](https://github.com/parse-community/Parse-SDK-Flutter/pull/762)
